src/Mvc/build/dependencies.props

```diff
@@ -34,8 +34,8 @@
     <MicrosoftAspNetCoreHostingAbstractionsPackageVersion>3.0.0-preview-18606-0099</MicrosoftAspNetCoreHostingAbstractionsPackageVersion>
     <MicrosoftAspNetCoreHostingPackageVersion>3.0.0-preview-18606-0099</MicrosoftAspNetCoreHostingPackageVersion>
     <MicrosoftAspNetCoreHtmlAbstractionsPackageVersion>3.0.0-preview-18606-0099</MicrosoftAspNetCoreHtmlAbstractionsPackageVersion>
-    <MicrosoftAspNetCoreHttpExtensionsPackageVersion>3.0.0-preview-18606-0099</MicrosoftAspNetCoreHttpExtensionsPackageVersion>
-    <MicrosoftAspNetCoreHttpPackageVersion>3.0.0-preview-18606-0099</MicrosoftAspNetCoreHttpPackageVersion>
+    <MicrosoftAspNetCoreHttpExtensionsPackageVersion>3.0.0-preview-18619-0194</MicrosoftAspNetCoreHttpExtensionsPackageVersion>
+    <MicrosoftAspNetCoreHttpPackageVersion>3.0.0-preview-18619-0194</MicrosoftAspNetCoreHttpPackageVersion>
```

Comment:
Everything usually moves in lock-step. Why are many packages left at `18606-0099`?

Reply:
run upgrade deps is broken and won't be fixed before MVC moves to `<reference>`. I could change everything to `18619-0194` but there would be depedencies with other versioning schemes that aren't the latest.

Think of this as a temporary thing until MVC is updated and does away with package references all together.